README.md

```diff
@@ -43,6 +43,7 @@ Read more about the package, and the intent behind it, in the [announcement on s
 - [`firstNonNil(_:)`](https://github.com/apple/swift-algorithms/blob/main/Guides/FirstNonNil.md): Returns the first non-`nil` result from transforming a sequence's elements.
 - [`indexed()`](https://github.com/apple/swift-algorithms/blob/main/Guides/Indexed.md): Iterate over tuples of a collection's indices and elements. 
 - [`interspersed(with:)`](https://github.com/apple/swift-algorithms/blob/main/Guides/Intersperse.md): Place a value between every two elements of a sequence.
+- [`partitioningIndex(where:)`](https://github.com/apple/swift-algorithms/blob/main/Guides/Partition.md)`: Splits a collection into two partitions according to a predicate and returns the index of the partition.
```

Comment:
@LemonSpike Sorry, hadn't actually read the text here! :facepalm:

`partitioningIndex` doesn't modify the collection, so let's go with something like this instead:

```suggestion
- [`partitioningIndex(where:)`](https://github.com/apple/swift-algorithms/blob/main/Guides/Partition.md)`: Returns the starting index of the partition of a collection that matches a predicate.
```